### ISSUE TITLE
Duplicate Headers

### ISSUE THREAD

**Issue by alexgoodey** [ebizmarts/sage-pay-suite-ce#24]
On line 168 of DirectPaymentController a header() is explicitly set. This causes a Duplicate Headers error in PHP-FPM when using Modal display for 3D secure.

Can you confirm this issue?

**Comment by martinebiz** [comment]
are you getting this on every order? is this affecting the payment process?

**Comment by alexgoodey** [comment]
Yes, this happens on every 3dcallback action. It throws an error in the iFrame on the modal and the user can progress no further. Note this is only using PHP-FPM (as mod_php does not complain about duplicate headers). If I change the line to use $this->_response->setHeader(....) then all works as expected.

**Comment by martinebiz** [comment]
Thanks for your feedback @alexgoodey, we'll see to check and introduce this fix for next release.
